self-host/docker-compose.yml

```diff
@@ -0,0 +1,153 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/compose-spec/compose-spec/master/schema/compose-spec.json
+
+# ^ this line provides nicer autocomplete in some editors
+
+services:
+    # the core platform service
+    platform:
+        depends_on:
+            db:
+                condition: service_started
+            platform-jobs:
+                condition: service_started
+            platform-migrations:
+                condition: service_completed_successfully
+        platform: linux/amd64
+        image: ghcr.io/pubpub/platform:latest
+        env_file:
+            - path: .env
+              required: true
+        environment:
+            DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+        ports:
+            - "3000:3000"
+        networks:
+            - app-network
+
+    # platfrom jobs service
+    # takes care of longer running tasks like scheduling actions
+    platform-jobs:
+        depends_on:
+            platform-migrations:
+                condition: service_completed_successfully
+        platform: linux/amd64
+        image: ghcr.io/pubpub/platform-jobs:latest
+        env_file:
+            - path: .env
+              required: true
+        environment:
+            DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+        networks:
+            - app-network
+
+    platform-migrations:
+        platform: linux/amd64
+        depends_on:
+            db:
+                condition: service_started
+        image: ghcr.io/pubpub/platform-migrations:latest
+        env_file:
+            - path: .env
+              required: true
+        environment:
+            DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+        # FIXME: remove this once it's merged into main
+        volumes:
+            - ../core/prisma/create-admin-user.cts:/usr/src/app/core/prisma/create-admin-user.cts
```

Comment:
this should be removed before merging, it's just here for testing.

reason: the `core/prisma/create-admin-user.cts` does not exist yet on `platform-migrations:latest`, because i just now added it 😅
once this pr is merged to main all will be swell